tests/testthat/test-validate.R

```diff
@@ -2,7 +2,7 @@
 test_that("Error is thrown if input values are of incorrect type", {
 
   # numeric
-  expect_error(validate_input_values(
+   expect_error(validate_input_values(
```

Comment:
nit: please tun automatic styler for every PR, this looks like an accidentally introduced whitespace

Reply:
styler done